resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtils.scala

```diff
@@ -88,9 +88,8 @@ private[spark] object KubernetesClientUtils extends Logging {
    * on remote pods.
    */
   def buildConfigMap(configMapName: String, confFileMap: Map[String, String],
-      withLabels: Map[String, String] = Map()): ConfigMap = {
-    val configMapNameSpace =
-      confFileMap.getOrElse(KUBERNETES_NAMESPACE.key, KUBERNETES_NAMESPACE.defaultValueString)
+                     configMapNameSpace: String,
```

Comment:
Hi，I don't think it is a good idea to change this method signature. Could you fix namespace in conFileMap?

Reply:
See my comments on https://github.com/apache/spark/pull/35215#discussion_r790218477